var/spack/repos/builtin/packages/py-gevent/package.py

```diff
@@ -11,26 +11,36 @@ class PyGevent(PythonPackage):
 
     homepage = "https://www.gevent.org"
     pypi = "gevent/gevent-1.3a2.tar.gz"
+    git = "https://github.com/gevent/gevent.git"
 
+    version("23.7.0", sha256="d0d3630674c1b344b256a298ab1ff43220f840b12af768131b5d74e485924237")
     version("21.12.0", sha256="f48b64578c367b91fa793bf8eaaaf4995cb93c8bc45860e473bf868070ad094e")
     version("21.8.0", sha256="43e93e1a4738c922a2416baf33f0afb0a20b22d3dba886720bc037cd02a98575")
     version("1.5.0", sha256="b2814258e3b3fb32786bb73af271ad31f51e1ac01f33b37426b66cb8491b4c29")
-    version("1.3a2", sha256="f7ab82697111ea233c7beeadf5240f669dfad9c4bbc89a3ec80a49e2c48a65bd")
+    version(
+        "1.3a2",
+        sha256="f7ab82697111ea233c7beeadf5240f669dfad9c4bbc89a3ec80a49e2c48a65bd",
+        deprecated=True,
+    )
+
+    # limits according to wheels available on PyPI
+    depends_on("python@3.8:3.12", when="@23.7.0:", type=("build", "run"))
+    depends_on("python@2.7:3.10", when="@21.8:21.12", type=("build", "run"))
+    depends_on("python@2.7:3.8", when="@1.5:20.6.0", type=("build", "run"))
+    depends_on("python@2.7:3.6", when="@1.3", type=("build", "run"))
```

Comment:
Let's just remove this version, `spack audit` fails because of this constraint.

Reply:
Done